alacritty.yml

```
@@ -278,9 +279,10 @@
   # Color used for the footer bar on the bottom, used by search regex input,
   # hyperlink URI preview, etc.
   #
+  # By default, these will use the opposing primary color.
```

```suggestion
  # By default, these will use the inverse primary color.
```

Since "inverse text" is the usual way terminal emulators refer to swapping foreground/background, I think it's good to use the well known language here.

---

I just copied the exact same line from the different option, so I'd change both of them.